singer-io/tap-recharge
# Rewrite tap with class approach

# Description of change
Tap has been rewritten with a class based approach for the streams as per [TDL-14804](https://jira.talendforge.org/browse/TDL-14804)

# Manual QA steps
 - ran tap manually
 - checked with `singer-check-tap`
 - ran with `target-stitch --dry-ryn` 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
